app/partials/actions/add.html

```diff
@@ -1,10 +1,10 @@
 <div class="container">
 	<h3>
-		Add {{rdesc.name.singular}}
+		Add {{rdesc.attributes.name.singular}}
 	</h3>
-	<h5>{{rdesc.description}}</h5>
+	<h5>{{rdesc.attributes.description}}</h5>
 
-	<div class="form-group" ng-repeat="field in rdesc.fields" ng-include="'/partials/fields/default-field.html'">
+	<div class="form-group" ng-repeat="field in rdesc.attributes.fields" ng-include="'/partials/fields/default-field.html'">
```

Comment:
Yeah this would be simple to do. I disagree with the naming though. I am partial to something like `constant`, `static`, or `automatic` for fields that can never be set by the user, and `readOnly` for fields that can only be set once. Maybe I'm biased from my days in C#, but the semantics they attach to the `readonly` keyword are the same as Java does with `final`, in that the value is set on object instantiation and then can never be reset, and I think that makes more sense.